setup.py

```diff
@@ -0,0 +1,23 @@
+from setuptools import find_packages, setup
+
+with open('requirements.txt') as f:
+    requirements=f.read().splitlines()
+
+setup(
+    name="spog-ui-tests",
+    version="0.0.1",
+    author="RHTC QE team",
```

Comment:
@mrrajan maybe we should change it to author="RHTAP QE team"?

Reply:
Ah, good catch. The script was created way back before.